src/d/actor/d_a_player_main.cpp

```diff
@@ -5082,7 +7607,343 @@ BOOL daPy_lk_c::checkAttentionPosAngle(fopAc_ac_c* actor, cXyz** pOutPos) {
+            fVar4 = std::abs(cM_scos(uVar18));
+            fVar3 = cM_ssin(uVar18);
+        } else if (mCurProc == daPyProc_CLIMB_MOVE_SIDE_e) {
+            fVar4 = -cM_scos(m34D4);
+            fVar3 = cM_ssin(m34D4);
+        } else if (((mCurProc == daPyProc_GRAB_HEAVY_WAIT_e) ||
+                    (mCurProc == daPyProc_GRAB_WAIT_e)) ||
+                   (mCurProc == daPyProc_WAIT_e))
+        {
+            fVar4 = 0.0f;
+            fVar3 = m35A0;
+        } else {
+            uVar18 = cM_atan2s(m36A0.x, m36A0.z) - shape_angle.y;
+            fVar4 = std::abs(cM_scos(uVar18));
+            fVar3 = cM_ssin(uVar18);
+        }
+        dVar22 = std::sqrtf(fVar3 * fVar3 + fVar4 * fVar4);
+        if (dVar22 > 1.0f) {
+            fVar1 = 1.0f / dVar22;
+            fVar4 = fVar4 * fVar1;
+            fVar3 = fVar3 * fVar1;
+        }
+        if (fVar3 > 0.0f) {
+            fVar2 = 0.12f;
+            in_f31 = fVar1 * fVar3;
+            fVar1 = -0.16f;
+            in_f30 = fVar2 * fVar3;
+        } else {
+            fVar2 = 0.16f;
+            in_f31 = fVar1 * fVar3;
+            fVar1 = -0.12f;
+            in_f30 = fVar2 * fVar3;
+        }
+        if (fVar4 > 0.0f) {
+            fVar3 = -0.12f;
+            in_f31 = fVar1 * fVar3;
+        } else {
+            fVar3 = -0.07f;
+            in_f30 = fVar2 * fVar3;
+        }
+        daPy_matAnm_c::setMorfFrame(0);
+        daPy_matAnm_c::onEyeMoveFlg();
+    } else {
+        sVar10 = m351C;
+        fVar4 = sVar10 / 4096.0f;
+
+        if (fVar4 > 1.0f) {
+            fVar1 = 1.0f;
+        } else if (fVar4 < -1.0f) {
+            fVar1 = -1.0f;
+        }
+        if (((!checkNoResetFlg1(daPyFlg1_CONFUSE)) && (!checkNoResetFlg1(daPyFlg1_UNK1000000))) &&
+            (sVar10 != 0))
+        {
+            if (sVar10 > 0) {
+                in_f31 = -0.16f * fVar1;
+                in_f30 = 0.12f * fVar1;
+                daPy_matAnm_c::onEyeMoveFlg();
+            } else {
+                in_f31 = -0.12f * fVar1;
+                in_f30 = 0.16f * fVar1;
+                daPy_matAnm_c::onEyeMoveFlg();
+            }
+        } else {
+            m_tex_eye_scroll[0]->mEyePos.x = 0.0f;
+            m_tex_eye_scroll[1]->mEyePos.x = 0.0f;
+            m_tex_eye_scroll[0]->mEyePos.y = 0.0f;
+            m_tex_eye_scroll[1]->mEyePos.y = 0.0f;
+            if (daPy_matAnm_c::getEyeMoveFlg() != 0) {
+                daPy_matAnm_c::setMorfFrame(3);
+            }
+            daPy_matAnm_c::offEyeMoveFlg();
+        }
+        fVar3 = 0.0f;
+    }
+    dVar22 = fVar3;
+    if (daPy_matAnm_c::getEyeMoveFlg() != 0) {
+        cLib_addCalc(&m_tex_eye_scroll[0]->mEyePos.x, in_f31, 0.5f, 0.1f, 0.03f);
+        cLib_addCalc(&m_tex_eye_scroll[1]->mEyePos.x, in_f30, 0.5f, 0.1f, 0.03f);
+        cLib_addCalc(&m_tex_eye_scroll[0]->mEyePos.y, dVar22, 0.5f, 0.08f, 0.02f);
+        m_tex_eye_scroll[1]->mEyePos.y = m_tex_eye_scroll[0]->mEyePos.y;
+    }
+    return;
 }
```

Comment:
```suggestion
    }
}
```
Explicit `return;` not needed at end of void func